lib/headcount_analyst.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "district_repository"
+
+class HeadcountAnalyst
+  attr_reader :district_repo
+
+  def initialize(district_repo)
+    @district_repo = district_repo
+  end
+
+  def load_district_repo_data
+    district_repo.load_data({
+      enrollment: {
+        kindergarten: "./data/Kindergartners in full-day program.csv"
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.